packages/components/src/unit-control/stories/index.story.tsx

```diff
@@ -59,6 +59,7 @@ export const Default: StoryFn< typeof UnitControl > = DefaultTemplate.bind(
 );
 Default.args = {
 	label: 'Label',
+	__next40pxDefaultSize: true,
```

Comment:
I'm also seeing UnitControl instances in ToolsPanel: the Storybook stories, the component JSDoc code snippet, and the readme.

Reply:
Thanks @mirka, I have updated the same.